README.md

```diff
@@ -115,6 +116,11 @@ versions for the best experience. These can be found
     `defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false`.
 -   To fix the remapped escape key not working in Linux, set `"keyboard.dispatch": "keyCode"`
 
+### Settings & Commands
+
+Please open the [vscode-neovim](vscode:extension/asvetliakov.vscode-neovim) extension details in VSCode, and check all
```

Comment:
Nit: this is phrased like a request, when really it's just informational

Reply:
Not good at writing in English

Reply:
That's ok. Let me try to craft something

Reply:
Changed

Reply:
> That's ok. Let me try to craft something

Feel free to modify this PR, I have no objection.